tests/testthat/test-population.R

```diff
@@ -0,0 +1,49 @@
+context("Population")
+populationFileName <- getTestDataFilePath("pop_10.csv")
+values <- c(1:10) * 2.5
+
+
+test_that("It can return the expected number of individual in the population", {
+  population <- loadPopulation(populationFileName)
+  expect_equal(population$count, 10)
+})
+
+test_that("It returns whether variability is defined for a parameter path", {
+  population <- loadPopulation(populationFileName)
+  expect_true(population$has("Organism|VenousBlood|Volume [l]"))

```

Comment:
I have updated the code so that it supports both query with and without units.
The reasoning behind that is that the path beign returned from a Quantity does not have the units. It would be very confusing for the user to ask `population$has("Organism|VenousBlood|Volume")` and get no as an answer.